qa/tasks/cephfs/filesystem.py

```diff
@@ -654,9 +662,10 @@ def create(self):
                 cmd.extend(self.ec_profile)
                 self.mon_manager.raw_cluster_cmd(*cmd)
                 self.mon_manager.raw_cluster_cmd(
-                    'osd', 'pool', 'create',
-                    ec_data_pool_name, self.pgs_per_fs_pool.__str__(), 'erasure',
-                    ec_data_pool_name)
+                    'osd', 'pool', 'create', ec_data_pool_name,
+                    'erasure', ec_data_pool_name,
+                    '--pg_num_min', self.pg_num_min.__str__(),
+                    '--target_size_ratio', self.target_size_ratio_ec.__str__())
```

Comment:
If the test is using an ec data pool, most file data should go there. The target size ratio should be `.9` in that case.

Reply:
Will fix it.
Thanks.